src/reflect/value.go

```diff
@@ -378,7 +378,31 @@ func (v Value) String() string {
 }
 
 func (v Value) Bytes() []byte {
-	panic("unimplemented: (reflect.Value).Bytes()")
+	switch v.Kind() {
+	case Slice:
+		if v.typecode.elem().Kind() != Uint8 {
+			panic(&ValueError{Method: "Bytes", Kind: v.Kind()})
+		}
+		hdr := *(*sliceHeader)(v.value)
+		return *(*[]byte)(unsafe.Pointer(&hdr))
+
+	case Array:
+		if v.typecode.elem().Kind() != Uint8 {
+			panic(&ValueError{Method: "Bytes", Kind: v.Kind()})
+		}
+
+		var hdr sliceHeader
+
+		l := uintptr(v.Len())
+
+		hdr.len = l
+		hdr.cap = l
+		hdr.data = v.value
+
+		return *(*[]byte)(unsafe.Pointer(&hdr))
```

Comment:
You can use `unsafe.Slice` for such an operation:

```suggestion
		return unsafe.Slice((*byte)(v.value), v.Len())
```